sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_shared/models.py

```diff
@@ -11,6 +11,7 @@
 
 
 class DeprecatedEnumMeta(CaseInsensitiveEnumMeta):
+
```

Comment:
nit extra space

Reply:
@l0lawrence I believe adding a newline after the class declaration before defining attributes or methods is perfectly acceptable. This practice enhances the readability and organization of the code, which is why we have followed this approach in the package. Do you have any concerns about this?

Reply:
Sorry I thought this was a typo